src/styles/_TransactionBuilder.scss

```diff
@@ -52,8 +52,19 @@
     text-align: center;
     font-size: $s-scale-a;
   }
+.TransactionOpMeta__duplicate {
+  text-align: center;
+  width: 72px;
+  padding: 0.35em 0;
+  font-size: $s-scale-down1;
+  margin: 0.25em 0;
+}
 .TransactionOpMeta__remove {
   text-align: center;
+  width: 72px;
+  padding: 0.35em 0;
+  font-size: $s-scale-down1;
+  margin: 0.25em 0;
 }
```

Comment:
These styles are exactly the same. We can create a more generic class for both buttons. What do you think? 🙂

Reply:
nice catch!